pkg/stack/unwind/compact_unwind_table.go

```diff
@@ -170,28 +179,12 @@ func rowToCompactRow(row *UnwindTableRow, arch elf.Machine) (CompactUnwindTableR
 	switch row.RBP.Rule {
 	case frame.RuleOffset:
 		rbpType = uint8(rbpRuleOffset)
-		// TODO(sylfrena): Reuse type DELET
-		// if arch == elf.EM_AARCH64 {
-		//	rbpType = uint8(fpRuleOffset) // Use one type here
-		// }
 		rbpOffset = int16(row.RBP.Offset)
-		// curious that the following condition doesn't satisfy. it should.
-		// On further investigation, it doesn't because only Offset Rule is applied, and register value is x0, not x29
-		// Ideally this whole thing should work with just rbpType = uint8(reusedrbp/fpRuleOffset)
-		// TODO(sylfrena): Delete this part later
-		// if row.RBP.Reg == frame.Arm64FramePointer && arch == elf.EM_AARCH64 {
-		// fmt.Println("Rule fp Offset(arm64)")
-		//	rbpType = uint8(fpRuleOffset)
-		// }
-	// TODO(sylfrena): Do these conditions also cover Arm64 DWARF?
```

Comment:
They do! It's the same for both architectures. The differences aren't in the DWARF itself but in the register mapping, the lr, etc